src/diffusers/pipelines/consistency_models/pipeline_consistency_decoder.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+import torch
+import torch.nn.functional as F
+
+from ...image_processor import VaeImageProcessor
+from ...models import UNet2DModel
+from ...schedulers.scheduling_consistency_decoder import ConsistencyDecoderScheduler
+from ...utils.torch_utils import randn_tensor
+from ..pipeline_utils import DiffusionPipeline, ImagePipelineOutput
+
+
+class ConsistencyDecoderPipeline(DiffusionPipeline):
+    model_cpu_offload_seq = "unet"
+
+    unet: UNet2DModel
+    scheduler: ConsistencyDecoderScheduler
+
+    def __init__(self, unet: UNet2DModel, scheduler: ConsistencyDecoderScheduler) -> None:
+        super().__init__()
+
+        self.register_modules(
+            unet=unet,
+            scheduler=scheduler,
+        )
+
+        self.image_processor = VaeImageProcessor()
+
+    @torch.no_grad()
+    def __call__(
+        self,
+        z,
+        generator=None,
+        output_type: Optional[str] = "pil",
+        num_inference_steps=2,
+        return_dict: bool = True,
+    ):
+        means = torch.tensor([0.38862467, 0.02253063, 0.07381133, -0.0171294], device=z.device, dtype=z.dtype)[
+            None, :, None, None
+        ]
+        stds = torch.tensor([0.9654121, 1.0440036, 0.76147926, 0.77022034], device=z.device, dtype=z.dtype)[
+            None, :, None, None
+        ]
```

Comment:
Wonder if it's better off as a config.

Reply:
was able to move to buffers on the vae now that it's a module